.github/workflows/build_and_test.yaml

```diff
@@ -218,6 +221,9 @@ jobs:
         run: |
           # Make sure to update this number this when adding or removing configmap or secrets
           test $(cat /tmp/logs/sidecar-pythonscript.log | grep "Hello from python script!" | wc -l) = "9"
+          test $(cat /tmp/logs/sidecar-pythonscript-logfile.log | grep "Hello from python script!" | wc -l) = "9"
```

Comment:
you'd need to concatenate this into a long single statement :see_no_evil: otherwise the exit code of the script does not reflect all assertions. See the other assertions.

Reply:
Can i use && for each test?

Reply:
exactly